safe/gui/tools/osm_downloader_dialog.py

```diff
@@ -97,7 +99,17 @@ def __init__(self, parent=None, iface=None):
         if proxy is not None:
             self.network_manager.setProxy(proxy)
         self.restore_state()
-        self.update_extent()
+
+        # Setup the rectangle map tool
+        self.canvas = iface.mapCanvas()
+        self.rectangle_map_tool = RectangleMapTool(self.canvas)
+        self.rectangle_map_tool.rectangle_created.connect(self.update_extent_from_rectangle)
```

Comment:
Hi @Gustry 

This line 106 and 107 are violating PEP8. Perhaps, you want to run `make pep8` first to see if there is some PEP8 violations.

Thanks